### PR TITLE
update doc for SPM

### PR DESCRIPTION
**Pull request by Reminiscent**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
In version5.2, we try to ban the feature for baseline evolution. So we need to forbid to open the switch.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
[TiDB pull#26333](https://github.com/pingcap/tidb/pull/26333)
[TiDB pull#26403](https://github.com/pingcap/tidb/pull/26403)
[TiDB pull#26340](https://github.com/pingcap/tidb/pull/26340)
[TiDB pull#26139](https://github.com/pingcap/tidb/pull/26139)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-plan-management.md [hunk 1]
```diff
@@ -187,6 +187,50 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+
+该语句使用系统变量 [last_plan_from_binding](https://docs.pingcap.com/zh/tidb/stable/system-variables#last_plan_from_binding-%E4%BB%8E-v40-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) 显示上一条执行的语句所使用的执行计划是否来自 binding 的执行计划。
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW [SESSION] STATUS LIKE 'last_plan_binding_update_time';
+```
+
+该语句使用状态变量 last_plan_binding_update_time 显示当前机器中 Binding 上一次的更新时间。其中 last_plan_binding_update_time 的默认值是 '0000-00-00 00:00:00'。
```
Comment (Reminiscent) [inline sql-plan-management.md:206]: 看了一下我们现在还没有专门的 status var 的文档，就先放在这个地方了。

Comment (eurekaka) [reply sql-plan-management.md:206]:
```suggestion
该语句使用状态变量 last_plan_binding_update_time 显示当前 tidb-server 上 Binding 缓存对应的时间戳。其中 last_plan_binding_update_time 的默认值是 '0000-00-00 00:00:00'。
```

其实不需要把 `last_plan_binding_update_time` 写到文档里，用户很难理解这个变量的意义，一般也只有出了问题的时候我们排查问题用。

File: sql-plan-management.md [hunk 2]
```diff
@@ -231,7 +275,7 @@ set global tidb_evolve_plan_baselines = on;
 
 > **注意：**
 >
-> 自动演进绑定功能目前不是 GA (Generally Available) 状态，不推荐在生产环境打开该功能。
+> 自动演进功能目前为实验特性，存在未知风险并需要继续优化，不建议在生产环境中使用。此开关将被禁用直到该功能到达 GA (Generally Available) 状态，如果尝试打开开关，会产生报错。如果你已经在生产环境中使用了此功能，请与 PingCAP 的技术支持联系获取相关支持。
```
Comment (eurekaka) [inline sql-plan-management.md:278]:
```suggestion
> 自动演进功能目前为实验特性，存在未知风险并需要继续优化，不建议在生产环境中使用。此开关将被禁用直到该功能到达 GA (Generally Available) 状态，如果尝试打开开关，会产生报错。如果你已经在生产环境中使用了此功能，请尽快将它关闭，如发现 Binding 状态不如预期，请与 PingCAP 的技术支持联系获取相关支持。
```

File: sql-plan-management.md (hunk 1 above)
Comment (Reminiscent) [reply sql-plan-management.md:206]: 那 `绑定的排查` 剩下的两点还需要加在里面么（一个是 explain + show warning 展示 binding，另一个是 last_plan_from_binding 这个系统变量）？

File: sql-plan-management.md [hunk 3]
```diff
@@ -187,6 +187,36 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+create global binding for
+    select * from t
+using
+    select * from t;
+
+-- 查看所有的 Binding
+show global bindings;
+
+-- 查看上一次 Binding 更新的时间，对比上一条语句的查询结果可以发现其展示的值是 Binding 最新更新的时间
+show status like 'last_plan_binding_update_time';
```
Comment (eurekaka) [inline sql-plan-management.md:213]: Please remove this as well.

File: sql-plan-management.md (hunk 1 above)
Comment (eurekaka) [reply sql-plan-management.md]: 需要吧

File: sql-plan-management.md [hunk 4]
```diff
@@ -165,6 +165,10 @@ explain select * from t1,t2 where t1.id = t2.id;
 
 在这里 SESSION 作用域内被删除掉的绑定会屏蔽 GLOBAL 作用域内相应的绑定，优化器不会为 `select` 语句添加 `sm_join(t1, t2)` hint，`explain` 给出的执行计划中最上层节点并不被 hint 固定为 MergeJoin，而是由优化器经过代价估算后自主进行选择。
 
+> **注意：**
+>
+> 每隔 100 个 `bind-info-lease`（默认值为 `3s`，合计 `300s`）后台的线程会触发一次对 update_time 在 10 个 `bind-info-lease` 以前的状态为 `deleted` 的 binding 的回收清除操作。
```
Comment (eurekaka) [inline sql-plan-management.md:170]:
```suggestion
> 执行 DROP GLOBAL BINDING 会将当前 tidb-server 缓存中的绑定删除，并将系统表中对应行的状态修改为 'deleted'。该命令不会直接将系统表记录删除的原因是，其他 tidb-server 需要读取这个 'deleted' 状态来删除其缓存中对应的绑定。对于这些系统表中状态为 'deleted' 的记录，后台线程每隔 100 个 `bind-info-lease`（默认值为 `3s`，合计 `300s`）会触发一次对 update_time 在 10 个 `bind-info-lease` 以前的绑定（确保所有 tidb-server 已经读去过这个 'deleted' 状态并更新完缓存）的回收清除操作。
```

File: sql-plan-management.md [hunk 5]
```diff
@@ -165,6 +165,10 @@ explain select * from t1,t2 where t1.id = t2.id;
 
 在这里 SESSION 作用域内被删除掉的绑定会屏蔽 GLOBAL 作用域内相应的绑定，优化器不会为 `select` 语句添加 `sm_join(t1, t2)` hint，`explain` 给出的执行计划中最上层节点并不被 hint 固定为 MergeJoin，而是由优化器经过代价估算后自主进行选择。
 
+> **注意：**
+>
+> 执行 DROP GLOBAL BINDING 会将当前 tidb-server 缓存中的绑定删除，并将系统表中对应行的状态修改为 'deleted'。该命令不会直接将系统表记录删除的原因是，其他 tidb-server 需要读取这个 'deleted' 状态来删除其缓存中对应的绑定。对于这些系统表中状态为 'deleted' 的记录，后台线程每隔 100 个 `bind-info-lease`（默认值为 `3s`，合计 `300s`）会触发一次对 update_time 在 10 个 `bind-info-lease` 以前的绑定（确保所有 tidb-server 已经读去过这个 'deleted' 状态并更新完缓存）的回收清除操作。
```
Comment (eurekaka) [inline sql-plan-management.md:170]:
```suggestion
> 执行 DROP GLOBAL BINDING 会将当前 tidb-server 缓存中的绑定删除，并将系统表中对应行的状态修改为 'deleted'。该命令不会直接将系统表记录删除的原因是，其他 tidb-server 需要读取这个 'deleted' 状态来删除其缓存中对应的绑定。对于这些系统表中状态为 'deleted' 的记录，后台线程每隔 100 个 `bind-info-lease`（默认值为 `3s`，合计 `300s`）会触发一次对 update_time 在 10 个 `bind-info-lease` 以前的绑定（确保所有 tidb-server 已经读取过这个 'deleted' 状态并更新完缓存）的回收清除操作。
```

File: sql-plan-management.md [hunk 6]
```diff
@@ -187,6 +191,30 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+
+```
+-- 创建一个 global binding
+create global binding for
+    select * from t
+using
+    select * from t;
+
+-- 使用 explain 语句查看 SQL 的执行计划，通过查看 warning 信息确认查询所使用的 Binding
+explain select * from t;
```
Comment (eurekaka) [inline sql-plan-management.md:214]: explain verbose?

File: sql-plan-management.md [hunk 7]
```diff
@@ -187,6 +191,30 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT [SESSION] @@last_plan_from_binding;
+```
+
+该语句使用系统变量 [last_plan_from_binding](https://docs.pingcap.com/zh/tidb/stable/system-variables#last_plan_from_binding-%E4%BB%8E-v40-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) 显示上一条执行的语句所使用的执行计划是否来自 binding 的执行计划。
+
+另外，当我们使用 explain 语句查看 SQL 的查询计划时，如果该 SQL 使用了 Binding，explain 语句会产生 warning，我们可以通过查看 warning 了解 SQL 使用了哪一条 Binding。
```
Comment (eurekaka) [inline sql-plan-management.md:204]: explain verbose?

**Review by eurekaka — APPROVED** [review]

**Review by pcqz — APPROVED** [review]

File: sql-plan-management.md [hunk 8]
```diff
@@ -165,6 +165,10 @@ explain select * from t1,t2 where t1.id = t2.id;
 
 在这里 SESSION 作用域内被删除掉的绑定会屏蔽 GLOBAL 作用域内相应的绑定，优化器不会为 `select` 语句添加 `sm_join(t1, t2)` hint，`explain` 给出的执行计划中最上层节点并不被 hint 固定为 MergeJoin，而是由优化器经过代价估算后自主进行选择。
 
+> **注意：**
+>
+> 执行 DROP GLOBAL BINDING 会将当前 tidb-server 缓存中的绑定删除，并将系统表中对应行的状态修改为 'deleted'。该命令不会直接将系统表记录删除的原因是，其他 tidb-server 需要读取这个 'deleted' 状态来删除其缓存中对应的绑定。对于这些系统表中状态为 'deleted' 的记录，后台线程每隔 100 个 `bind-info-lease`（默认值为 `3s`，合计 `300s`）会触发一次对 update_time 在 10 个 `bind-info-lease` 以前的绑定（确保所有 tidb-server 已经读取过这个 'deleted' 状态并更新完缓存）的回收清除操作。
```
Comment (TomShawn) [inline sql-plan-management.md:170]:
```suggestion
> 执行 `DROP GLOBAL BINDING` 会删除当前 tidb-server 实例缓存中的绑定，并将系统表中对应行的状态修改为 'deleted'。该语句不会直接删除系统表中的记录，因为其他 tidb-server 实例需要读取系统表中的 'deleted' 状态来删除其缓存中对应的绑定。对于这些系统表中状态为 'deleted' 的记录，后台线程每隔 100 个 `bind-info-lease`（默认值为 `3s`，合计 `300s`）会触发一次对 `update_time` 在 10 个 `bind-info-lease` 以前的绑定（确保所有 tidb-server 实例已经读取过这个 'deleted' 状态并更新完缓存）的回收清除操作。
```

File: sql-plan-management.md [hunk 9]
```diff
@@ -187,6 +191,31 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
 | collation | 排序规则 |
 | source | 创建方式，包括 manual （由 `create [global] binding` 生成）、capture（由 tidb 自动创建生成）和 evolve （由 tidb 自动演进生成） |
 
+### 绑定的排查
```
Comment (TomShawn) [inline sql-plan-management.md:194]:
```suggestion
### 排查绑定
```

File: sql-plan-management.md [hunk 10]
```diff
@@ -187,6 +191,31 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT [SESSION] @@last_plan_from_binding;
+```
+
+该语句使用系统变量 [last_plan_from_binding](https://docs.pingcap.com/zh/tidb/stable/system-variables#last_plan_from_binding-%E4%BB%8E-v40-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) 显示上一条执行的语句所使用的执行计划是否来自 binding 的执行计划。
```
Comment (TomShawn) [inline sql-plan-management.md:202]:
```suggestion
该语句使用系统变量 [`last_plan_from_binding`](/system-variables.md#last_plan_from_binding-从-v40-版本开始引入) 显示上一条执行的语句所使用的执行计划是否来自 binding 的执行计划。
```

File: sql-plan-management.md [hunk 11]
```diff
@@ -187,6 +191,31 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+
+另外，当我们使用 explain format = 'verbose'  语句查看 SQL 的查询计划时，如果该 SQL 使用了 binding，explain 语句会产生 warning，我们可以通过查看 warning 了解 SQL 使用了哪一条 binding。
```
Comment (TomShawn) [inline sql-plan-management.md:204]:
```suggestion
另外，当使用 `explain format = 'verbose'`  语句查看一条 SQL 语句的查询计划时，如果该 SQL 语句使用了 binding，`explain` 语句会输出 warning 警告。此时可以通过查看 warning 了解该 SQL 语句使用了哪一条 binding。
```

File: sql-plan-management.md [hunk 12]
```diff
@@ -187,6 +191,31 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+-- 创建一个 global binding
+create global binding for
+    select * from t
+using
+    select * from t;
+
+-- 使用 explain format = 'verbose' 
+语句查看 SQL 的执行计划，通过查看 warning 信息确认查询所使用的 binding
+explain format = 'verbose' select * from t;
+show warnings;
```
Comment (TomShawn) [inline sql-plan-management.md:216]:
```suggestion
-- 创建一个 global binding

create global binding for
    select * from t
using
    select * from t;

-- 使用 explain format = 'verbose' 语句查看 SQL 的执行计划，通过查看 warning 信息确认查询所使用的 binding

explain format = 'verbose' select * from t;
show warnings;
```

File: sql-plan-management.md [hunk 13]
```diff
@@ -231,7 +260,7 @@ set global tidb_evolve_plan_baselines = on;
 
 > **注意：**
 >
-> 自动演进绑定功能目前不是 GA (Generally Available) 状态，不推荐在生产环境打开该功能。
+> 自动演进功能目前为实验特性，存在未知风险并需要继续优化，不建议在生产环境中使用。此开关将被禁用直到该功能到达 GA (Generally Available) 状态，如果尝试打开开关，会产生报错。如果你已经在生产环境中使用了此功能，请尽快将它关闭，如发现 Binding 状态不如预期，请与 PingCAP 的技术支持联系获取相关支持。
```
Comment (TomShawn) [inline sql-plan-management.md:263]: [PingCAP 的技术支持] 是否有联系方式？
```suggestion
> 自动演进功能目前为实验特性，存在未知风险，不建议在生产环境中使用。
>
> 此变量开关已强制关闭，直到该自动演进成为正式功能 GA (Generally Available)。如果你尝试打开开关，会产生报错。如果你已经在生产环境中使用了此功能，请尽快将它禁用。如发现 binding 状态不如预期，请与 PingCAP 的技术支持联系获取相关支持。
```

File: sql-plan-management.md [hunk 14]
```diff
@@ -231,7 +260,7 @@ set global tidb_evolve_plan_baselines = on;
 
 > **注意：**
```
Comment (TomShawn) [inline sql-plan-management.md:261]:
```suggestion
> **警告：**
```

File: sql-plan-management.md [hunk 15]
```diff
@@ -229,9 +259,11 @@ set global tidb_evolve_plan_baselines = on;
 
 `tidb_evolve_plan_baselines` 的默认值为 `off`。
 
-> **注意：**
+> **警告：**
+>
+> 自动演进功能目前为实验特性，存在未知风险，不建议在生产环境中使用。
 >
-> 自动演进绑定功能目前不是 GA (Generally Available) 状态，不推荐在生产环境打开该功能。
+> 此变量开关已强制关闭，直到该自动演进成为正式功能 GA (Generally Available)。如果你尝试打开开关，会产生报错。如果你已经在生产环境中使用了此功能，请尽快将它禁用。如发现 binding 状态不如预期，请与 PingCAP 的技术支持联系获取相关支持。
```
Comment (TomShawn) [inline sql-plan-management.md:266]:
```suggestion
> 此变量开关已强制关闭，直到自动演进成为正式功能 GA (Generally Available)。如果你尝试打开开关，会产生报错。如果你已经在生产环境中使用了此功能，请尽快将它禁用。如发现 binding 状态不如预期，请与 PingCAP 的技术支持联系获取相关支持。
```

File: sql-plan-management.md [hunk 16]
```diff
@@ -187,6 +191,32 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+
+```sql
+SELECT [SESSION] @@last_plan_from_binding;
+```
+
+该语句使用系统变量 [`last_plan_from_binding`](/system-variables.md#last_plan_from_binding-从-v40-版本开始引入) 显示上一条执行的语句所使用的执行计划是否来自 binding 的执行计划。
+
+另外，当使用 `explain format = 'verbose'`  语句查看一条 SQL 语句的查询计划时，如果该 SQL 语句使用了 binding，`explain` 语句会输出 warning 警告。此时可以通过查看 warning 了解该 SQL 语句使用了哪一条 binding。
+
+```
```
Comment (TomShawn) [inline sql-plan-management.md:206]:
```suggestion
```sql
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: sql-plan-management.md [hunk 17]
```diff
@@ -187,6 +191,32 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhere];
...
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT [SESSION] @@last_plan_from_binding;
+```
+
+该语句使用系统变量 [`last_plan_from_binding`](/system-variables.md#last_plan_from_binding-从-v40-版本开始引入) 显示上一条执行的语句所使用的执行计划是否来自 binding 的执行计划。
+
+另外，当使用 `explain format = 'verbose'`  语句查看一条 SQL 语句的查询计划时，如果该 SQL 语句使用了 binding，`explain` 语句会输出 warning 警告。此时可以通过查看 warning 了解该 SQL 语句使用了哪一条 binding。
```
Comment (TomShawn) [inline sql-plan-management.md:204]:
```suggestion
另外，当使用 `explain format = 'verbose'` 语句查看一条 SQL 语句的查询计划时，如果该 SQL 语句使用了 binding，`explain` 语句会输出 warning 警告。此时可以通过查看 warning 了解该 SQL 语句使用了哪一条 binding。
```

**Pull request merged**